pwa/src/components/entities/entityForm.tsx

```diff
@@ -137,7 +137,7 @@ export const EntityForm: React.FC<EntityFormProps> = ({entityId}) => {
                   </button>
                 </Link>
                 <button
-                  className="utrecht-button utrec`ht-button-sm btn-sm btn-success"
+                  className="utrecht-button utrecht`ht-button-sm btn-sm btn-success"
```

Comment:
Should be `utrecht-button-sm`. I see you fixed one of the typos but left another one in.